pkg/providers/launchtemplate/suite_test.go

```diff
@@ -120,9 +120,9 @@ func ExpectLaunchTemplatesCreatedWithUserDataContaining(substrings ...string) {
 	Expect(awsEnv.EC2API.CalledWithCreateLaunchTemplateInput.Len()).To(BeNumerically(">=", 1))
 	awsEnv.EC2API.CalledWithCreateLaunchTemplateInput.ForEach(func(input *ec2.CreateLaunchTemplateInput) {
 		userData, err := base64.StdEncoding.DecodeString(*input.LaunchTemplateData.UserData)
-		ExpectWithOffset(2, err).To(BeNil())
```

Comment:
We need these offsets here since this ForEach is actually double nested due to the function itself as well as the function that is called here. I didn't find a good way to do this with `GinkgoHelper` so I think the current approach is reasonable.